tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "resolveJsonModule": true,
     "experimentalDecorators": true,
     "strictNullChecks": true,
+    "skipLibCheck": true,
```

Comment:
The primary reason for this is that it avoids some errors thrown by third-party .d.ts libraries with TypeScript 3.7. It has the beneficial side effect of speeding up the build (by not spending time type-checking third-party .d.ts files). Since we can't do anything about type errors in third-party .d.ts files anyway, I'm inclined to bank the build time reduction even once the .d.ts errors are fixed.